ui/page/home/view.jsx

```diff
@@ -158,7 +167,7 @@ function HomePage(props: Props) {
           {title && typeof title === 'string' && (
             <div className="homePage-wrapper__section-title">
               <SectionHeader title={__(resolveTitleOverride(title))} navigate={route || link} icon={icon} help={help} />
-              {index === 0 && <CustomizeHomepage />}
+              {id === 'FOLLOWING' && <CustomizeHomepage />}
```

Comment:
With this change you can only customize if you have the `Following` section in your feed (aside from the customize toggle in the sidebar) but you can hide it and `<CustomizeHomepage />` is also hidden, or you can put it at the bottom and have to scroll every time to change, so since not every one will have Following at the top, is this intended or is Following supposed to stay "pinned" at the top?

Reply:
Good point. I've replaced the 0 because 0 is now the banner by default. We can't show it above the banner. However, my solution seems kinda stupid now that I think about your comment. We might need something entirely new. Maybe something that looks more like the current cog icon on mobile.

Reply:
Fixed.